src/rgw/setup/rgw_service.py

```diff
@@ -27,8 +28,9 @@ class RgwStart:
     def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
-        except OSError as e:
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
+            cmd = shlex.split(cmd)
+            os.execl(cmd)
+        except Exception as e:
```

Comment:
Why have we removed OSError ? Can we include that
except Exception catches only generic errors and we do not know What type of OSError it is. 

For example: If /usr/bin/radosgw executable is not found/installed. It will throw error stating 'Failed to start radosgw service'. But it should throw something like File/executable missing.

Reply:
@selvakumaar5496 
For that we have printed the exceotion in the log file itself.
**Log.error(f"Failed to start radosgw service:{e}")**
Above line in logs will tell what exception is found.

Reply:
Okay. But It if it some other exception, It will not have attribute e.errno

Reply:
Separated OSError with Exception.